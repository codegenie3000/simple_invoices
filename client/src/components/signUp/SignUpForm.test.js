import React from 'react';
import {shallow, mount} from 'enzyme';

import SignUpFormRedux, {SignUpForm} from './SignUpForm';

describe('<SignUpFormRedux/>', ()=> {
    let component;
    beforeAll(()=> {
        const handleSubmit = value => {console.log('submit handled')};
        const connectForm = value => {console.log('contact form prop')};
        component = mount(<SignUpForm handleSubmit={handleSubmit} connectForm={connectForm}/>);
    });
    it('matches snapshot', ()=> {
        expect(component).toMatchSnapshot();
    });
});