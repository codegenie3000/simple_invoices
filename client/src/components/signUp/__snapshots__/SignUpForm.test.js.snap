// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpFormRedux/> matches snapshot 1`] = `
<div
  className="containerWhiteRounded"
>
  <div
    className="flexOneOuter"
  >
    <div
      className="flexOneInner"
    >
      <h1>
        Sign Up
      </h1>
      <form>
        <Fields />
        <ButtonPrimary
          buttonText="Sign up"
        />
      </form>
    </div>
  </div>
</div>
`;
